lib/clean.py

```diff
@@ -790,3 +790,152 @@ def convert_dates(df: pd.DataFrame, cols: list[str]) -> pd.DataFrame:
+                r"^E jefferson levee pd", "East Jefferson Levee PD", regex=True
+            )
+            .str.replace(r"^E\b ", "", regex=True)
+            .str.replace(r"^Univ PDxavier", "Xavier University PD", regex=True)
+            .str.replace(r"La State Police", "Louisiana State PD", regex=True)
+            .str.replace(r"^Univ PDlsuhscno$", "LSUHSC - No University PD", regex=True)
+            .str.replace(
+                r"^Univ PDdelgado Cc$",
+                "Delgado Community College University PD",
+                regex=True,
+            )
+            .str.replace(r"Univ PDdillard", "Dillard University PD", regex=True)
+            .str.replace(r"^Outstate", "Out of State", regex=True)
+            .str.replace(
+                r"^Medical Centerlano$", "Medical Center Of La - No", regex=True
+            )
+            .str.replace(r"^Univ PDloyola$", "Loyola University PD", regex=True)
+            .str.replace(r"^Univ PDlsu$", "LSU University PD", regex=True)
+            .str.replace(r"^Orleans", "New Orleans", regex=True)
+            .str.replace(r"\bLsu\b", "LSU", regex=True)
+            .str.replace(
+                r"Univ PDsouthernno",
+                "Southern University PD - New Orleans ",
+                regex=True,
+            )
+            .str.replace(r" Par ", "", regex=True)
+            .str.replace(
+                r"Alcoholtobacco Control", "Alcohol Tobacco Control", regex=True
+            )
+            .str.replace(
+                r"^Housing Authorityno$", "Housing Authority of New Orleans", regex=True
+            )
+            .str.replace(r"^Univ PDtulane$", "Tulane University PD", regex=True)
+            .str.replace(r"Univ PDuno", "UNO University PD")
+            .str.replace(r"\bLa\b", "Louisiana")
+            .str.replace(r"^PlaqueminesSO$", "Plaquemines SO", regex=True)
+            .str.replace(r"\bNo\b", "New Orleans PD", regex=True)
+            .str.replace(r"^St\. Martin$", "St. Martin SO")
+            .str.replace(r"^Tangipahoa$", "Tangipahoa SO", regex=True)
+            .str.replace(r"^Lake Charles$", "Lake Charles PD")
+            .str.replace(r"^St\. Tammany$", "St. Tammany SO", regex=True)
+            .str.replace(r"^Hammond$", "Hammond PD", regex=True)
+            .str.replace(r"^New Orleans$", "New Orleans PD")
+            .str.replace(r"^Lafayette City$", "Lafayette City PD", regex=True)
+            .str.replace(r"^Grand Isle$", "Grand Isle PD", regex=True)
+            .str.replace(r"^New Orleans Levee$", "New Orleans Levee PD", regex=True)
+            .str.replace(r"^Harbor PD$", "New Orleans Harbor PD", regex=True)
+            .str.replace(r"^Lafayette$", "Lafayette PD", regex=True)
+            .str.replace(r"^Louisiana State$", "Louisiana State PD", regex=True)
+            .str.replace(r"^Caddo Parish$", "Caddo SO", regex=True)
+        )
+
+    return df
+
+
+def clean_post_agency_row(df: pd.DataFrame, cols: list[str]) -> pd.DataFrame:
```

Comment:
The `clean_post_agency_*` functions do not feel like they belong in `lib/clean.py` because they are not reused outside of `post_history.py`.

Reply:
Please remove redundant functions from this file.